sync/downstream.py

```diff
@@ -181,14 +181,14 @@ def next_action(self) -> DownstreamAction:
         if not self.requires_try:
             return DownstreamAction.ready
         if self.error:
-            return DownstreamAction.manual_fix
+            self.try_rebase()
```

Comment:
Presumably if the rebase fails we want to return `DownstreamAction.manual_fix` at this point, but if it succeeds we want to continue with the remainder of the function (similarly below).